osci.yaml

```diff
@@ -3,20 +3,9 @@
       - charm-unit-jobs
     check:
       jobs:
-      - impish-xena:
-          voting: false
-      - hirsute-wallaby
-      - groovy-victoria
-      - focal-xena:
-          voting: false
-      - focal-wallaby
-      - focal-victoria
-      - focal-ussuri
-      - bionic-ussuri
       - bionic-queens
-      - bionic-stein
-      - bionic-train
-      - bionic-rocky
+      - focal-ussuri
```

Comment:
please add jammy-yoga and focal-yoga as well. I know we don't have the hardware, but ...